docs/src/getting-started.rst

```diff
@@ -59,14 +69,17 @@ Initializing the Project
 
 .. note::
 
-    This guide walks you through the process of creating a |cdk| project
-    step-by-step to explain some of the reasoning and details
-    behind the project structure and tools. It is also possible to use the
-    :code:`cdk init` command to get started quickly from a project
-    template in supported languages.
+    This guide walks you through the process of creating an |cdk| project.
+    You can also use the
+    :code:`cdk init` command to create a skeleton project from a
+    template in any of the supported languages except JavaScript.
```

Comment:
That's going to be outdated soon. Maybe we can just write "in supported languages".

Reply:
done